nrrd/reader.py

```diff
@@ -78,8 +78,8 @@ def _get_field_type(field, custom_field_map):
         return 'double list'
     elif field in ['kinds', 'labels', 'units', 'space units', 'centerings']:
         return 'string list'
-    elif field in []:
-        return 'int vector'
+    # elif field in []:
+    #     return 'int vector'
     elif field in ['space origin']:
         return 'double vector'
     elif field in ['measurement frame']:
```

Comment:
Isn't this the place where Isaiah and I corrected 'double matrix'? Not sure if you are working on a different branch.

Reply:
Yeah, this is the place. I just checked this PR and it's up to date. The `measurement frame` field type is 'double matrix' as corrected.